smac/intensifier/intensifier.py

```diff
@@ -81,7 +81,11 @@ def uses_instances(self) -> bool:  # noqa: D102
 
     def get_state(self) -> dict[str, Any]:  # noqa: D102
         return {
-            "queue": [(self.runhistory.get_config_id(config), n) for config, n in self._queue],
+            "queue": [
+                (self.runhistory.get_config_id(config), n)
+                for config, n in self._queue
+                if self.runhistory.has_config(config)
```

Comment:
So now you basically just ignore the config?

Reply:
They are the configs that are put in the queue but not executed yet. Currently, our RH cannot (and do not necessarily) record the potential future configurations, I think it is fine to simply drop them.